content/modules/rest/pages/metadata-api.adoc

```diff
@@ -0,0 +1,202 @@
+= Metadata API
+
+It is possible to introspect the entity structure and relationships between entities in Jmix via the xref:data-model:metadata.adoc[Metadata] mechanism. You can also access these capabilities through the REST API with the different metadata endpoints starting with `/metadata`.
```

Comment:
"You can access these capabilities through the REST API endpoints starting with `/metadata`."